Nuget/ILMerge.MSBuild.Task.nuspec

```diff
@@ -1,21 +1,24 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
-  <metadata>
-    <id>ILMerge.MSBuild.Task</id>
-    <version>$version$</version>
-    <title>ILMerge Configurable MSBuild Task</title>
-    <authors>Emerson Brito</authors>
-    <owners>Emerson Brito</owners>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Adds ILMerge to Visual Studio 2013/2017 or automated builds. This Task is intended to work right out of the box however an optional configuration file can be used to cover virtually all ILMerge properties including the list of assemblies to be merged and output directory.</description>    
-    <copyright>Emerson Brito (c) 2016</copyright>
-    <projectUrl>https://github.com/emerbrito/ILMerge-MSBuild-Task</projectUrl>
-	<license type="expression">MIT</license>	
-    <tags>ilmerge msbuild dynamics crm 365 merge</tags>
-  </metadata>
-  <files>
-    <file src="..\ILMerge.MSBuild.Task\ILMerge.MSBuild.Task\bin\release\ILMerge.MsBuild.Task.dll" target="tools\ILMerge.MsBuild.Task.dll" />
-    <file src="..\ILMerge.MSBuild.Task\ILMerge.MSBuild.Task\bin\release\ILMerge.MsBuild.Task.pdb" target="tools\ILMerge.MsBuild.Task.pdb" />
-    <file src="..\ILMerge.MSBuild.Task\ILMerge.MSBuild.Task\build\ILMerge.MSBuild.Task.targets" target="build\ILMerge.MSBuild.Task.targets" />
-  </files>
-</package>
\ No newline at end of file
+	<metadata>
+		<id>ILMerge.MSBuild.Task</id>
+		<version>$version$</version>
+		<title>ILMerge Configurable MSBuild Task</title>
+		<authors>Emerson Brito</authors>
+		<owners>Emerson Brito</owners>
+		<requireLicenseAcceptance>false</requireLicenseAcceptance>
+		<description>Adds ILMerge to Visual Studio 2013/2017 or automated builds. This Task is intended to work right out of the box however an optional configuration file can be used to cover virtually all ILMerge properties including the list of assemblies to be merged and output directory.</description>    
+		<copyright>Emerson Brito (c) 2016</copyright>
+		<projectUrl>https://github.com/emerbrito/ILMerge-MSBuild-Task</projectUrl>
+		<license type="expression">MIT</license>	
+		<tags>ilmerge msbuild dynamics crm 365 merge</tags>
+		<dependencies>
+			<dependency id="ilmerge" version="(,3.0.29)" />
```

Comment:
adding the dependency is a good idea, but specifying a version is questionable.

having said that, `version="(,3.0.29)"` means **up to but excluding** version **3.0.29** - which is the latest version and it does work; we do not want to exlude it.  this should otherwise be `(,3.0.29]`, but again.. why specify a version in the first place?  unless there is a breaking change in the latest version (to come) we shouldn't be specifying a maximum at all.